index.js

```diff
@@ -14,6 +14,28 @@ async function maybeResponseLfs(response) {
 	}
 }
 
+async function repoListingSlashblanchSupport(ref, dir, repoListingConfig) {
+	let files;
+	const dirParts = decodeURIComponent(dir).split('/');
+	while (dirParts.length >= 0) {
+		try {
+			files = await listContent.viaTreesApi(repoListingConfig); // eslint-disable-line no-await-in-loop
+			break;
+		} catch {
```

Comment:
I realized that the API might fail for a number of reasons, so the `catch` here should only allow the loop for this specific 404 error